scripts/bootstrap/buildbootstrapcli.sh

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+usage()
+{
+    echo "Builds a bootstrap CLI from sources"
+    echo "Usage: $0 [BuildType] --rid <Rid> --seedcli <SeedCli> [--arch <Architecture>] [--os <OS>] [--clang <Major.Minor>] [--corelib <CoreLib>] [--crossgen]"
```

Comment:
`--crossgen` doesn't appear to be wired up yet

Reply:
Thanks for spotting that.